tests/gocase/util/random.go

```diff
@@ -71,7 +75,7 @@ func RandString(min, max int, typ RandStringType) string {
 
 	var sb strings.Builder
 	for ; length > 0; length-- {
-		s := fmt.Sprintf("%c", minVal+rand.Intn(maxVal-minVal+1))
+		s := fmt.Sprintf("%c", minVal+int(r.Int31n(int32(maxVal-minVal+1))))
```

Comment:
Here is used to avoid random seed in `rand.Intn`, it would be difficult to debug with machine's random seed.